tests/e2e_tests/kgain_e2e.py

```diff
@@ -10,13 +10,86 @@
 import corgidrp.walker as walker
 import corgidrp.caldb as caldb
 
+try:
+    from cal.kgain.calibrate_kgain import calibrate_kgain
+    import cal
+except:
+    pass
+
 thisfile_dir = os.path.dirname(__file__) # this file's folder
 
-tvac_kgain = 8.8145 #e/DN, result from new iit drp code
-tvac_readnoise = 130.12 #e, result from new iit drp code
+# tvac_kgain = 8.49404981510777 #8.8145 #e/DN, result from new iit code with specified file input order
```

Comment:
can these lines be deleted?

Reply:
Well, I kind of like having a record of the current results just for comparison when another user tries the test, and I like having the numerical results before the change as well (i.e., the second set of numbers) in case some future change makes a reversion back to those results somehow.  I just made them into more readable comments that don't seem like they are meant to be uncommented.  If you disagree, I can delete the lines.

Reply:
ok, as long as they are purposeful, that's fine